Program.cs

```diff
@@ -1,7 +1,47 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿// --------------------------------------------------------------------------------------------
+// Training ~ A training program for new joinees at Metamation, Batch - July 2023
+// Copyright (c) Metamation India.                                              
+// ------------------------------------------------------------------------
+// Program.cs
+// STRING PERMUTATIONS
+// Generate all string character permutations. 
+// For example, Input: "NOT", Output: "NOT NTO ONT OTN TNO TON"
+// --------------------------------------------------------------------------------------------
+namespace Training {
+   #region Program ------------------------------------------------------------------------------
+   /// <summary>String permutation</summary>
+   internal class Program {
+      #region Methods ---------------------------------------------
+      /// <summary>Displays all string string permutation</summary>
+      /// <param name="args">arguments</param>
+      static void Main (string[] args) {
+         string word = "NOT";
+         Permute (word.ToCharArray (), 0);
+      }
+
+      /// <summary>Permutate the given word and print all string permutation</summary>
+      /// <param name="wordArr">Char array</param>
+      /// <param name="fixedChar">index of char</param>

```

Comment:
Description of fixedChar is not very clear. It just says index of char. Which char index should I pass?